### PR TITLE
add views.py, Serializer, urls.py(use Router)

### PR DESCRIPTION
**Pull request by yury-rubanenko**
1) Add an API view to display the list of words within the "platforms"  (done)

2) Describe the serializer using the Word and UserWord models. (done)

3) Define views using the serializer and create class-based views to display both the complete word list and the user's word list.(done)

4) Define API-style paths in the urls.py file for the "Words" and "My Words" views.(done)

5) The "My Words" view can be categorized and searched using tags. You can use the query parameter "status=/" for this purpose. (done)

### CODE REVIEW

File: Language_Project/urls.py [hunk 1]
```diff
@@ -4,4 +4,5 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/v1/profiles/', include('profiles.urls')),
+    path('api/v2/platforms/', include('platforms.urls')),
```
Comment (ArezKhalimi) [inline Language_Project/urls.py]: it's still v1 .
1. We use REST approach 
2. use DRF with the same version
3. payload is the same
4. no clients - no one use previous implementation
no reason to mark it as a new version

Comment (ArezKhalimi) [reply Language_Project/urls.py]: good naming. well done

File: platforms/serializers.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from rest_framework import serializers
+from .models import UserWord, Word
+
+
+class UserWordSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = UserWord
+        fields = ('__all__')
```
Comment (ArezKhalimi) [inline platforms/serializers.py]: never use the "lazy" definition for payload
describe a list of fields what you plan to use here

File: platforms/serializers.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+class WordSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Word
+        fields = ('__all__')
```
Comment (ArezKhalimi) [inline platforms/serializers.py]: same as above

File: platforms/views.py [hunk 4]
```diff
@@ -1,3 +1,28 @@
...
+from .models import UserWord, Word
+from .serializers import UserWordSerializer, WordSerializer
+
+class UserWordsListView(viewsets.ReadOnlyModelViewSet):
+    serializer_class = UserWordSerializer
+    permission_classes = [permissions.IsAuthenticated]
+
+    def get_queryset(self):
+        user = self.request.user
+        status = self.request.query_params.get('status', None)
```
Comment (ArezKhalimi) [inline platforms/views.py]: use DRF-filters for this

**Review by ArezKhalimi — APPROVED** [review]

**Review by arezafrosiab — APPROVED** [review]

**Pull request merged**